app/cdap/services/ThemeHelper.ts

```diff
@@ -223,6 +224,7 @@ interface IThemeObj {
   featureNames?: IFeatureNames;
   showSqlPipeline?: boolean;
   showCDC?: boolean;
+  isMetadataInReact?: boolean;
```

Comment:
Can we add a todo here? Eventually we want the feature flags to be in a context provider situation and I don't think keeping it in themes makes sense long term.